src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_FileIO_Native.wasm.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.Storage;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_Storage
+{
+	[TestClass]
+	public class Given_FileIO_Native : Given_FileIO_Native_Base
+	{
+		protected override async Task<StorageFolder> GetRootFolderAsync()
+		{
+			var folder = await StorageFolder.GetPrivateRootAsync()!;
+			if (folder == null)
+			{
+				Assert.Inconclusive("File System Access API not available on this browser.");
```

Comment:
Do you know if this actually works on the CI?

Reply:
Yes it does :-) . I made some adjustments to the `UnitTestControl` to pick the `AssertInconclusiveException` so the test then shows among the "Ignored" tests.